### PR TITLE
minor spelling changes in protocol.md

### PR DESCRIPTION
**Pull request by yannickark**

### CODE REVIEW

File: doc/protocol.md [hunk 1]
```diff
@@ -123,23 +123,23 @@ S: SVR GAME <player result> {PLAYERONESCORE: "<score player1>", PLAYERTWOSCORE:
...
 S: SVR GAME CHALLENGE CANCELLED {CHALLENGENUMBER: "<challange number>"}  
 ->Challange has expired. Possible causes: player started another challange, player started a match or the player disconnected from the game.  
 
-**Accepting a challange:**  
-C: challenge accept <challange number>  
+**Accepting a challenge:**  
+C: challenge accept <challenge number>  
 S: OK  
-->The challange has been excepted. The match will be started, message will follow.  
+->The challange has been accepted. The match will be started, message will follow.
```
Comment (roffel) [inline doc/protocol.md]: You've missed another challange.

File: doc/protocol.md [hunk 2]
```diff
@@ -123,23 +123,23 @@ S: SVR GAME <player result> {PLAYERONESCORE: "<score player1>", PLAYERTWOSCORE:
...
 
-**Receiving a challange:**  
+**Receiving a challenge:**  
 S: SVR GAME CHALLENGE {CHALLENGER: <player>, GAMETYPE: <gametype>, CHALLENGENUMBER: <challangenumber>}  
-->Now the possibility to accept the challange.  
+->Now the possibility to accept the challenge.  
 
 **Result of a challange that has expired:**  
 S: SVR GAME CHALLENGE CANCELLED {CHALLENGENUMBER: "<challange number>"}  
 ->Challange has expired. Possible causes: player started another challange, player started a match or the player disconnected from the game.
```
Comment (roffel) [inline doc/protocol.md]: Challange. Maybe just do a replace all. :stuck_out_tongue:

Comment (roffel) [inline doc/protocol.md]: Challange. Maybe just do a replace all. :stuck_out_tongue:

Comment (yannickark) [inline doc/protocol.md]: lol

**Comment by goto-bus-stop** [comment]
(You can keep working on the same branch after you've opened a pull request, then the pull request will be updated automatically.)

**Pull request closed without merging**